### PR TITLE
Add metadata for fdroid

### LINKED ISSUE #5: FR: App on f-droid
Hey,
I would like to see this app f-droid, an foss app store for android. I think its easier to distribute (for people using fdroid) than downloading from github.

### PR DESCRIPTION
**Pull request by mueller-ma**
Fixes #5

### CODE REVIEW

File: fastlane/metadata/android/de-DE/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Die ist eine Android Anwendung zur Vollbilddarstellung der HABPanel Oberfläche von openHAB (einem open-source Projekt zur Hausautomation).
```
Comment (mueller-ma) [inline fastlane/metadata/android/de-DE/full_description.txt]: "Android-Anwendung"

File: fastlane/metadata/android/de-DE/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Sie ermöglicht eine Vollbilddarstellung der Steuerungsoberfläche, und kann als System Launcher eingestellt werden damit sie automatisch mit dem Gerät gestartet wird.
```
Comment (mueller-ma) [inline fastlane/metadata/android/de-DE/full_description.txt]: No comma, "System-Launcher", "werden, damit"

File: fastlane/metadata/android/de-DE/full_description.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+Weitere Funktionen sind: automatisches Erkennen des openHAB Servers im lokalen Netz, Home Screen/Launcher, Sensoren des Gerätes können überwacht, und deren Werte an openHAB übermittelt werden. Die Kamera des Gerätes kann zur Bewegungserkennung verwendet werden. Der Bildschirm des Gerätes, das Kamera Blitzlicht und die Gerätelautstärke können durch openHAB gesteuert werden.
```
Comment (mueller-ma) [inline fastlane/metadata/android/de-DE/full_description.txt]: "openHAB-Servers", launcher function already mentioned, no comma before "und", "Kamerablitz"

File: fastlane/metadata/android/de-DE/short_description.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Eine Android Anwendung zur Vollbilddarstellung der HABPanel UI von openHAB
```
Comment (mueller-ma) [inline fastlane/metadata/android/de-DE/short_description.txt]: "Android-Anwendung"

File: fastlane/metadata/android/de-DE/title.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+HABPanelViewer Kiosk browser für HABPanel openHAB
```
Comment (mueller-ma) [inline fastlane/metadata/android/de-DE/title.txt]: "für openHABs HABPanel"

File: fastlane/metadata/android/en-US/title.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+HABPanelViewer Kiosk utility for openHAB HABPanel
```
Comment (mueller-ma) [inline fastlane/metadata/android/en-US/title.txt]: "openHAB's HABPanel"?

File: fastlane/metadata/android/en-US/full_description.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+This is a native Android application for openHAB's HABPanel dashboard user interface - an open source home automation system.
+
+This application is indended to provide a customized kiosk mode for the HABPanel user interface, a part of the openHAB platform. www.openhab.org
+
+Features include: Auto-discovery of your openHAB server, Home Screen/Launcher, Kiosk Mode, Integration to use and set openHAB "Items", Motion detection, Several screen control options, Volume control, flashlight control, ability to report various tablet sensor/statues back to openHAB Items, many more!
```
Comment (mueller-ma) [inline fastlane/metadata/android/en-US/full_description.txt]: A list would suit better than a long sentence.

**Comment by mueller-ma** [comment]
I already changed some "OpenHAB" to "openHAB", but not in the ods file.

**Comment by vbier** [comment]
All these changes are okay. Do you add them to your branch so I can just merge them, or how do we do this?

**Comment by mueller-ma** [comment]
Yes, I can make them

**Comment by mueller-ma** [comment]
Done, you may also want to do that for the play store description.

**Comment by vbier** [comment]
I have updated the play store description. Thanks for your help on this. 

Maybe you want to announce the availability in the forum thread once the app can be installed from fdroid?

**Pull request merged**